src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -31,3 +31,5 @@ export const classes = [
   'Test',
   'Procedure',
 ];
+
+export const INTERNAL_MAPPING_DEFAULT_SOURCE = 'CIEL';
```

Comment:
Could you explain a bit why you have prefixed this with internal mapping?

Reply:
I prefixed it with internal mapping because Andrew Kanter said that when the source of a concept is CIEL (for now), it should be considered an internal concept.